src/RockSample.jl

```diff
@@ -55,6 +66,16 @@ function RockSamplePOMDP(map_size,
                              )
 end
 
+function rsgen(map)
```

Comment:
Could you document this function? It seems like it is generating a random instance of the rocksample problem. 
I have two comments: 
- add a docstring 
- maybe add an rng as an optional argument

Reply:
Actually, I am wondering if this function should not override the constructor rather: 
`RockSamplePOMDP(map, rng=GLOBAL_RNG)` instead of `rsgen`?